src/components/MainDashboard/PrescriptionsList/PrescriptionsList.module.css

```diff
@@ -0,0 +1,65 @@
+.prescriptions-list-div p {
+  margin: 0;
+}
+
+.prescriptions-list-div {
+  background-color: #d9d9d9;
+  display: inline-block;
+  gap: 10px;
+  padding: 10px 20px;
+  border-radius: 10px;
+  margin: 20px;
+  max-height: 35rem;
+  overflow-x: visible;
+  overflow-y: auto;
+}
+
+.prescriptions-list-item {
+  background-color: white;
+  height: 5rem;
+  width: 15rem; /* Fixed width for each item */
+  padding: 15px;
+  border-radius: 10px;
+  overflow: hidden;
+  color: #284b63;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.prescriptions-list-item span {
+  overflow: hidden;
+  color: #284b63;
+  text-align: justify;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-family: Inter;
```

Comment:
The font-family is not imported and that's why it's not applied. You should import it as 
`@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');` at the top of the css file.
I recommend checking out Google Fonts and how to use them